scalableminds/webknossos
# Add tooltips for settings in left sidebar

### URL of deployed dev instance (used for testing):
- https://moretooltips.webknossos.xyz

### Steps to test:
- open view mode, annotation mode (orthogonal, flight), with and without volume
- hover over settings labels, tooltips should be explanatory

### Issues:
- fixes #5566 

------
